pkgs/servers/scylladb/default.nix

```diff
@@ -75,28 +103,25 @@ gcc8Stdenv.mkDerivation {
    thrift
   ];
 
-  postPatch = ''
-    patchShebangs ./configure.py
-    patchShebangs seastar/json/json2code.py
-  '';
-
   configurePhase = ''
-    ./configure.py --mode=release
+    echo -e '#!/bin/sh\ncat $NIX_CC/nix-support/dynamic-linker' > reloc/get-dynamic-linker.sh
+
+    patchShebangs --build .
+
+    ./configure.py --enable-dpdk --mode=release
   '';
 
   installPhase = ''
-    mkdir $out
-    cp -r * $out/
+    install -D -m555 -t $out/bin/scylla build/release/scylla
+    install -D -m555 -t $out/bin/iotune build/release/iotune
```

Comment:
Both `scylla` and `iotune` are single binaries, do they need their own directories?